sdk/python/pyproject.toml

```diff
@@ -64,3 +64,8 @@ core-metadata-version = "2.3"
 # This ensures that the version for pulumi in uv.lock is updated when the
 # version from _version changes.
 upgrade-package = ["pulumi"]
+
+[tool.pyright]
+"venvPath" = ".venv"
```

Comment:
@julienp the Zed Python docs have the virtual environment settings split over 2 properties:

```toml
[tool.pyright]
venvPath = "."
venv = ".venv"
```

https://zed.dev/docs/languages/python#virtual-environments

The PyRight docs as well:

https://github.com/microsoft/pyright/blob/main/docs/configuration.md

Reply:
Thanks, I'll update that!